source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -0,0 +1,552 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Features.Scripts;
+using Calamari.Common.Plumbing;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Proxies;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Kubernetes.Authentication;
+using Calamari.Kubernetes.Integration;
+using Newtonsoft.Json.Linq;
+using Octopus.CoreUtilities;
+using Octopus.Versioning.Semver;
+
+namespace Calamari.Kubernetes
+{
+    public class SetupKubectlAuthentication
```

Comment:
This was removed from `KubernetesContextScriptWrapper.cs` so that it can be used outside of a "run script" context.